lib/error.js

```diff
@@ -84,27 +84,29 @@ export const makeError = ({
 	error.shortMessage = shortMessage;
 	error.command = command;
 	error.escapedCommand = escapedCommand;
+	error.cwd = cwd;
```

Comment:
The command's request properties (`command` and `cwd`) come before the result ones (`exitCode`, etc.).